src/pages/RequestInvite/index.tsx

```diff
@@ -44,9 +43,9 @@ const RequestInviteElem = styled.main`
 `;
 
 interface State {
-  videoUrl: string | null; // Will contain video auth token, for this client only do not store
+  videoUrl: string | null | undefined; // Will contain video auth token, for this client only do not store
```

Comment:
Not totally sure why this stuff is potentially both null or undefined, but if so you can use `void` instead of `null | undefined`. That said I'd prefer if we could just make these kinds of things undefined and then use optional syntax like `videoUrl?: string`.

Reply:
hmm, I tried using `void` but I'm told that `'null' is not assignable to type 'string | void'`.

Reply:
ah, this explains it (https://github.com/Microsoft/TypeScript/issues/8322). I think our return types are a little messy. I'll leave a todo to address in a later PR.

My inclination is to just use `undefined` and stop using `null` to indicate that a value is, well, undefined lol. Does that sound reasonable to you?